Src/Add-AzureDtlVM.ps1

```diff
@@ -1,64 +1,142 @@
+<#
+.SYNOPSIS 
+    This script adds the specified number of Azure virtual machines in the specified DevTest Lab.
+
+.DESCRIPTION
+    It allows the creation of the VMs inside a specific lab. This script needs to be scheduled for the creation of the VMs of each lab. 
```

Comment:
Please expand on the description. I.E. 'It allows to specify most parameters of interest for a lab creation (i.e. image name, network name, etc...). It should be a paragraph with one to three phrases. Please do this for all the other descriptions of functions in the source files.